node/src/main.rs

```diff
@@ -39,14 +45,47 @@ use graph_server_websocket::SubscriptionServer as GraphQLSubscriptionServer;
 use graph_store_postgres::{Store as DieselStore, StoreConfig};
 
 fn main() {
+    let (shutdown_sender, shutdown_receiver) = oneshot::channel();
+    // Register guarded panic logger which ensures logs flush on shutdown
     let (panic_logger, _panic_guard) = guarded_logger();
-    register_panic_hook(panic_logger);
-    tokio::run(future::lazy(async_main))
+    register_panic_hook(panic_logger, shutdown_sender);
+
+    // Create components for tokio context: multi-threaded runtime, reactor reference,
+    // executor context on the runtime, and Timer handle.
+    let runtime = tokio::runtime::Runtime::new().expect("Failed to create runtime");
+    let reactor = tokio_reactor::Handle::current();
```

Comment:
Is this variable unused? Looks like the whole `tokio_reactor` dependency is unused now.

Reply:
yep 👍, I've now removed the entire `tokio_reactor` dependency.